spec/services/checksum_validator_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe ChecksumValidator do
+  it 'has some behaviors'
```

Comment:
wonder if this should be a skip to show up as "needs work" rather than a false pass??

Reply:
@ndushay: fixed!